packages/database/src/migrations/20220920062403-AddEntityIdIndexOnSurveyResponseTable-modifies-schema.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = async function (db) {
+  await db.runSql(`CREATE INDEX survey_response_entity_id_idx ON survey_response (entity_id);`);
+  await db.runSql(`CREATE INDEX survey_response_outdated_id_idx ON survey_response (outdated);`);
+  await db.runSql(
+    `CREATE INDEX survey_response_data_time_idx ON survey_response (data_time DESC);`,
+  );
```

Comment:
We're kinda building up an 'index per column` pattern on the `survey_response` table... I don't know if that's a database anti-pattern or not... but it's hard to know how we will want to query the table...

Reply:
Emm and even we have `user_id` index, it still takes some times to load in admin panel when searching by user name, that could relates to the join table though.